src/domains/Invite/InviteForm/CreateInvite.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useCallback } from 'react';
+import Loader from '@material-ui/core/CircularProgress';
+import Grid from '@material-ui/core/Grid';
+import useEndpoint from 'hooks/useEndpoint';
+import API from '../api';
+import { InviteForm, InvitePreview } from '../types';
+
+interface Props {
+    onSuccess: () => void;
+    onFailure: () => void;
+    inviteForm: InviteForm;
+    file: File | undefined;
+    preview: InvitePreview;
+}
+
+export default function CreateInvite({
```

Comment:
I moved this to steps. The primary reason for keeping it as a component is simply for the loader to be displayed while the request is submitted, otherwise, the step would just be blank and might be confusing to users with longer loading times.